cmd/kubeadm/app/phases/addons/dns/dns_test.go

```diff
@@ -550,3 +553,230 @@ func createClientAndCoreDNSManifest(t *testing.T, corefile, coreDNSVersion strin
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			out := &bytes.Buffer{}
+			if err := coreDNSAddon(tt.args.cfg, tt.args.client, &tt.args.replicas, out, tt.args.printManifest); (err != nil) != tt.wantErr {
+				t.Errorf("coreDNSAddon() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+			if gotOut := out.String(); gotOut != tt.wantOut {
+				t.Errorf("coreDNSAddon() = %v, wantOut %v", gotOut, tt.wantOut)
```

Comment:
```suggestion
				t.Errorf("Actual output of coreDNSAddon() does not match expected.\nActual:  %v\nExpected: %v\n", gotOut, tt.wantOut)
```
As the output is mutliline, better to add `\n` before and after.